### ISSUE TITLE
Incorrect values in `view["stat"]["funds"]` record

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#124]
Bug in this code:
```
                    # e.g. {"rub": {"total": 10000.99, "totalCostRUB": 10000.99, "free": 1234.56, "freeCostRUB": 1234.56}, "usd": {"total": 250.55, "totalCostRUB": 15375.80, "free": 125.05, "freeCostRUB": 7687.50}}
                    view["stat"]["funds"][currency] = {
                        "total": volume,
                        "totalCostRUB": costRUB,  # total volume cost in rubles
                        "free": volume - blocked,
                        "freeCostRUB": costRUB * ((volume - blocked) / volume) if volume > 0 else 0,  # free volume cost in rubles
                    }
```

Incorrect result:
```
    "_funds": {
        "usd": {
            "total": 0.34,
            "totalCostRUB": 73.78,
            "free": 0.34,
            "freeCostRUB": 73.78
        },
        "rub": {
            "total": 163168.29,
            "totalCostRUB": 163168.29,
            "free": 163168.29,
            "freeCostRUB": 163168.29
        },
        "eur": {
            "total": 0.22,
            "totalCostRUB": 79.26,
            "free": 0.22,
            "freeCostRUB": 79.26
        },
        "hkd": {
            "total": 17006.5,
            "totalCostRUB": 9.55,
            "free": 17000.29,
            "freeCostRUB": 9.55
        }
    }
```

**Comment by Tim55667757** [comment]
Fixed in build v1.6.dev147

Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/260631707
PyPI srtefact: https://pypi.org/project/tksbrokerapi/1.6.dev147/

**Comment by Tim55667757** [comment]
Fixed in build v1.6.dev148

Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/260636223
PyPI srtefact: https://pypi.org/project/tksbrokerapi/1.6.dev148/

**Comment by Tim55667757** [comment]
Oh, it seems be good now:

![image](https://user-images.githubusercontent.com/7660870/219026798-9b8def26-2aed-4e82-b68c-8988830e3fcb.png)

Fixed in build v1.6.dev151

Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/260670872
PyPI artefact for testing: https://pypi.org/project/tksbrokerapi/1.6.dev151/
